FGD/tf-brokk.fgd

```diff
@@ -1842,7 +1842,7 @@ target(target_destination) : "Entity to point at. Will overwrite the entity's an
 
 	maxpropscreenwidth(float) : "Start Fade Pixels" : -1 : "Number of pixels wide at which all props in the level start to fade (<0 = use End Fade Pixels). This number is ignored if the prop has a specific fade distance specified."
 	minpropscreenwidth(float) : "End Fade Pixels" : 0 : "Minimum number of pixels wide at which the prop is visible (0 = don't fade out). This number is ignored if the prop has a specific fade distance specified."
-	maxoccluderarea(float) : "Max occluder Area" : "0.0" : "Prevents occlusion testing for models that take up more than X% of the screen."
+	maxoccludeearea(float) : "Max occluder Area" : "0.0" : "Prevents occlusion testing for models that take up more than X% of the screen."
```

Comment:
You didn't fully undo the previous change. It should say "Max Occludee Area", as in "the maximum screen area of that which is occluded".